cypress/integration/user/bookings_list.js

```diff
@@ -0,0 +1,99 @@
+describe('List bookings', () => {
+    before(async () => {
+      await cy.exec('dotenv -e .env.test -- npx prisma migrate reset --force')
+    })
+  
+    it('Should not have permissions when not logged in', () => {
+      cy.visit('http://localhost:3000/booking/list')
+      
+      cy.contains('You do not have enough permissions for this action!')
+    })
+  
+    it('Should have permissions when logged in as user', () => {
+      cy.login('normal@example.com', 'password')
+      cy.visit('http://localhost:3000/booking/list')
+      
+      cy.contains('Name: Booking 2' +
+        'Phone: 0213522341' + 
+        'Mail: booking2@gmail.com' +
+        'Starts at: 10.02.2017, 20:00:00 Ends at: 10.02.2028, 21:00:00')
+      cy.contains('Name: Booking 1' +
+        'Phone: 0213212341' +
+        'Mail: booking1@gmail.com' +    
+        'Starts at: 10.02.2019, 20:00:00 Ends at: 10.02.2020, 21:00:00')
+      cy.contains('You do not have enough permissions for this action!').should('not.exist')
+      cy.contains('No bookings!').should('not.exist')
+    })
+
+    it('Should have permissions when logged in as admin', () => {
+      cy.login('admin@example.com', 'password')
+      cy.visit('http://localhost:3000/booking/list')
+      
+      cy.contains('Name: Booking 2' +
```

Comment:
let's extract those to a function checkBookingsPresent so we don't repeat the code